cli/formatter/format_test.go

```diff
@@ -0,0 +1,277 @@
+	}
+
+	for _, c := range cases {
+		t.Run(c.expected, func(t *testing.T) {
+			if c.panic {
+				f := func() { _ = c.format.String() }
+				assert.PanicsWithValue(t, "Unknown output format", f)
+			} else {
+				result := c.format.String()
+				assert.Equal(t, c.expected, result, "Unexpected result")
+			}
+		})
+	}
+}
+
+func TestFormatter_MakeOutputFormat(t *testing.T) {
+	cases := []struct {
+		outputFormat formatter.Format
+		input        string // string from server side
+		output       string // render content to console
+		panic        bool
+	}{
+		{
+			// when user typed to console: localhost:xxxx> 1,2,3
+			formatter.DefaultFormat,
+			"---\n- 1\n- 2\n- 3\n...",
+			"---\n- 1\n- 2\n- 3\n...\n",
+			false,
+		},
+		{
+			// when user typed to console: localhost:xxxx> 1,2,3
+			formatter.YamlFormat,
+			"---\n- 1\n- 2\n- 3\n...",
+			"---\n- 1\n- 2\n- 3\n...\n",
+			false,
+		},
+		{
+			// when user typed to console: localhost:xxxx> 1,2,3
+			formatter.LuaFormat,
+			"--- [1, 2, 3]\n...",
+			"1, 2, 3;\n",
+			false,
+		},
+		{
+			// when user typed to console: localhost:xxxx> 1,2,3
+			formatter.TableFormat,
+			"--- [1, 2, 3]\n...",
+			"+------+\n" +
+				"| col1 |\n" +
+				"+------+\n" +
+				"| 1    |\n" +
+				"+------+\n" +
+				"| 2    |\n" +
+				"+------+\n" +
+				"| 3    |\n" +
+				"+------+\n",
+			false,
+		},
+		{
+			// when user typed to console: localhost:xxxx> 1,2,3
+			formatter.TTableFormat,
+			"--- [1, 2, 3]\n...",
+			"+------+---+---+---+\n" +
+				"| col1 | 1 | 2 | 3 |\n" +
+				"+------+---+---+---+\n",
+			false,
+		},
+		{
+			// when user typed to console:
+			// localhost:xxxx> box.tuple.new({1,2,'hello'}), box.tuple.new({10,20})
+			formatter.DefaultFormat,
+			"---\n- [1, 2, 'hello']\n- [10, 20]\n...",
+			"---\n- [1, 2, 'hello']\n- [10, 20]\n...\n",
+			false,
+		},
+		{
+			// when user typed to console:
+			// localhost:xxxx> box.tuple.new({1,2,'hello'}), box.tuple.new({10,20})
+			formatter.YamlFormat,
+			"---\n- [1, 2, 'hello']\n- [10, 20]\n...",
+			"---\n- [1, 2, 'hello']\n- [10, 20]\n...\n",
+			false,
+		},
+		{
+			// when user typed to console:
+			// localhost:xxxx> box.tuple.new({1,2,'hello'}), box.tuple.new({10,20})
+			formatter.TableFormat,
+			"---\n- [1, 2, 'hello']\n- [10, 20]\n...",
+			"+------+------+-------+\n" +
+				"| col1 | col2 | col3  |\n" +
+				"+------+------+-------+\n" +
+				"| 1    | 2    | hello |\n" +
+				"+------+------+-------+\n" +
+				"| 10   | 20   |       |\n" +
+				"+------+------+-------+\n",
+			false,
+		},
+		{
+			// when user typed to console:
+			// localhost:xxxx> box.tuple.new({1,2,'hello'}), box.tuple.new({10,20})
+			formatter.TTableFormat,
+			"---\n- [1, 2, 'hello']\n- [10, 20]\n...",
+			"+------+-------+----+\n" +
+				"| col1 | 1     | 10 |\n" +
+				"+------+-------+----+\n" +
+				"| col2 | 2     | 20 |\n" +
+				"+------+-------+----+\n" +
+				"| col3 | hello |    |\n" +
+				"+------+-------+----+\n",
+			false,
+		},
+		{
```

Comment:
Could you please add samples for TableFormat and TTableFormat with names?

https://github.com/tarantool/tt/blob/a4647e8ac6e4871f8239dbba4734fd8ccff22e3e/test/integration/connect/test_connect.py#L897-L908

Reply:
Added